src/data/multiset/basic.lean

```diff
@@ -377,6 +377,8 @@ ne_of_gt (lt_cons_self _ _)
 ⟨λ h, mem_of_le h (mem_singleton_self _),
  λ h, let ⟨t, e⟩ := exists_cons_of_mem h in e.symm ▸ cons_le_cons _ (zero_le _)⟩
 
+theorem pair_comm (x y : α) : ({x, y} : multiset α) = {y, x} := cons_swap x y ∅
```

Comment:
For `multiset` the idiomatic way to write `∅` is `0` (maybe it's simp normal form too?)

Reply:
Oh right, I was actually the one who made it simp normal 🤦🏻. Will fix in a followup PR.

Reply:
It's in a proof though, so it doesn't actually matter  :)
[multiset.induction_on'](https://leanprover-community.github.io/mathlib_docs/data/multiset/basic.html#multiset.induction_on') is the only declaration in multiset.basic that still has a ∅.